packages/dataviews/src/field-types/stories/index.story.tsx

```diff
@@ -94,6 +97,8 @@ const data: DataType[] = [
 		emailWithElements: 'hi@example.com',
 		telephone: '+1-555-123-4567',
 		telephoneWithElements: '+1-555-123-4567',
+		color: '#ff6600',
+		colorWithElements: 'rgba(255, 102, 0, 0.8)',
```

Comment:
```suggestion
		colorWithElements: 'rgba(255, 165, 0, 0.8)',
```

so the initial value is one of the elements